### PR TITLE
EVG-2310 discrete monitor logging

### PR DESCRIPTION
**Pull request by john-m-liu**

### CODE REVIEW

File: monitor/host_monitoring.go [hunk 1]
```diff
@@ -25,7 +26,11 @@ type hostMonitoringFunc func(*evergreen.Settings) []error
 // monitorReachability is a hostMonitoringFunc responsible for seeing if
 // hosts are reachable or not. returns a slice of any errors that occur
 func monitorReachability(settings *evergreen.Settings) []error {
-	grip.Info("Running reachability checks...")
+	grip.Info(message.Fields{
+		"runner":   RunnerName,
+		"function": "monitorReachability",
```
Comment (bsamek) [inline monitor/host_monitoring.go]: We use "operation" elsewhere. We should probably pick one.

File: monitor/tasks.go [hunk 2]
```diff
@@ -50,28 +50,34 @@ func (tm *TaskMonitor) CleanupTasks(ctx context.Context, projects map[string]mod
...
 }
 
 // clean up the passed-in slice of tasks
 func cleanUpTasks(taskWrappers []doomedTaskWrapper, projects map[string]model.Project) []error {
-	grip.Infof("Cleaning up %d tasks...", len(taskWrappers))
+	grip.Info(message.Fields{
+		"runner":  RunnerName,
+		"message": "Cleaning up tasks",
+		"count":   len(taskWrappers),
+		"tasks":   taskWrappers,
```
Comment (bsamek) [inline monitor/tasks.go]: Could this be potentially very large? Should we avoid logging it?

File: monitor/host_monitoring.go (hunk 1 above)
Comment (john-m-liu) [reply monitor/host_monitoring.go]: changed

File: monitor/tasks.go (hunk 2 above)
Comment (john-m-liu) [reply monitor/tasks.go]: oops, thought the struct was just 2 strings

**Pull request merged**